src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1748,174 +1748,164 @@ def _ec_key_set_public_key_affine_coordinates(self, ctx, x, y):
         return ctx
 
     def _private_key_bytes(self, encoding, format, encryption_algorithm,
-                           evp_pkey, cdata):
+                           key, evp_pkey, cdata):
```

Comment:
You added key to args here but I can't find where it's used. Am I missing it?

Reply:
It's used by OpenSSH PR...

Reply:
Generally speaking, any serializer that does not use OpenSSL directly probably wants regular instances.